src/fairseq2/models/conformer/convolution.py

```diff
@@ -74,7 +78,22 @@ def __init__(
             dtype=dtype,
         )
 
-        self.batch_norm = BatchNorm1d(model_dim, device=device, dtype=dtype)
+        if normalization not in ("batch_norm", "layer_norm"):
+            raise ValueError(
+                "We only support batch_norm and layer_norm within the `ConformerConvolution` module."
+            )
+
+        if normalization == "batch_norm":
+            self.batch_norm = BatchNorm1d(model_dim, device=device, dtype=dtype)
+        else:
+            self.register_module("batch_norm", None)
+
+        if normalization == "layer_norm":
+            self.layer_norm = create_default_layer_norm(
```

Comment:
Since you do not take a `layer_norm_fn` parameter (which I think is not necessary since Hirofumi's architecture always uses `StandardLayerNorm`), you can directly instantiate a `StandardLayerNorm` here instead of this factory function.